src/cdf.rs

```diff
@@ -710,6 +710,49 @@ use crate::src::levels::N_MV_JOINTS;
 use crate::include::common::intops::imin;
 use crate::src::levels::N_COMP_INTER_PRED_MODES;
 use crate::src::r#ref::dav1d_ref_inc;
+const fn cdf0d<const P: usize, const N: usize>(probs: [u16; P]) -> [u16; N] {
+    assert!(P < N);
+    let mut cdf0d = [0; N];
+    let mut i = 0;
+    while i < P {
+        cdf0d[i] = (32768 - probs[i] as u32) as u16;
```

Comment:
```suggestion
        cdf0d[i] = probs[i].wrapping_neg();
```

Reply:
It isn't clear to me that using `wrapping_neg` is equivalent to the previous computation. Perhaps that's why the comment shows as outdated.

Reply:
Yeah, I mixed up the range of u16 and i16. It turns out the conversions are unnecessary.

Reply:
I'm considering this small change in behavior so that the probability range is enforced and zero values map back to zero. I think that where my code currently differs from the original C code.